evap/evaluation/templates/contact_modal.html

```diff
@@ -1,3 +1,4 @@
+{% load static %}
```

Comment:
We usually have an extra line here:

```suggestion
{% load static %}

```